charts/spire/values.yaml

```diff
@@ -24,6 +24,42 @@ server:
     accessMode: ReadWriteOnce
     storageClass: null
 
+  service:
+    type: ClusterIP
+    port: 8081
+
+oidc:
+  enabled: false
+
+  image:
+    repository: gcr.io/spiffe-io/oidc-discovery-provider
+    pullPolicy: IfNotPresent
+    tag: ""
```

Comment:
Should we have a default tag here?

Reply:
Same holds for server above

Reply:
default it takes the version mentioned in Chart.yaml. this allows to override the chart wide appVersion.